### DISCUSSION TITLE
Support Avalonia officially

### DISCUSSION THREAD

**Discussion by OmidID** [PrismLibrary/Prism#2713] [General]
Hello, 
I would like to ask you if you agree to move the [Prism.Avalonia](https://github.com/AvaloniaCommunity/Prism.Avalonia) project here and be part of Prism officially.

We recently migrated it to v8 and I imagine it is a good time to have it supported here.

Thanks,

**Comment by dansiegel** [comment]
While from a high level I'm not directly opposed to the concept, for @brianlagunas and I this is not a winning proposition to take on the Avalonia codebase and suddenly find ourselves responsible. Neither of us work with Avalonia or have clients who are using it which means there is no real incentive for us, it just becomes a drain on our time, energy and financial resources to maintain. If the Avalonia team would like to have private discussions on how they can ease that burden then we're certainly willing to discuss that the same as we did with the Uno team.

**Reply by DamianSuess** [reply]
@OmidID and @dansiegel I'm open to having the discussion with you guys 👍  It would be nice to sync up with you guys to see the best way to serve the community.

Thanks, Dan, for getting back.

**Reply by dansiegel** [reply]
feel free to email at dsiegel@avantipoint.com or ping me on discord

**Reply by danwalmsley** [reply]
@DamianSuess id be happy to join in discussion, as I'm sure Avalonia can help maintain and support this.

dan@avaloniaui.net

**Reply by OmidID** [reply]
Thank you all. I am so glad for this advantage. :)

**Reply by DamianSuess** [reply]
Time flies when you're having fun.. Well, more-so working to meet quarterly goals.
@dansiegel and @OmidID, did you guys happen to meet up in regards to this at all?

**Reply by OmidID** [reply]
@dansiegel What is the next step?

**Reply by dansiegel** [reply]
You might start with opening an email thread with details around what you're proposing. 😉

**Reply by DamianSuess** [reply]
Sounds good, thank you Dan.
